### PR TITLE
Add ros2_control metapackage

### PR DESCRIPTION
**Pull request by piraka9011**
Add a metapackage for ros2_control.

Signed-off-by: Anas Abou Allaban <aabouallaban@pm.me>

### CODE REVIEW

**Comment by bmagyar** [comment]
Do you know what could be wrong with the CI? Missing rosdep keys should have been flagged up when those changes were merged in the first place!

**Comment by Karsten1987** [comment]
also before merging this, I'd request for all people mentioned here to approve this PR.

**Comment by piraka9011** [comment]
@Karsten1987 friendly ping!

File: ros2_control/package.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  <exec_depend>controller_interface</exec_depend>
+  <exec_depend>controller_manager</exec_depend>
+  <exec_depend>controller_parameter_server</exec_depend>
+  <exec_depend>hardware_interface</exec_depend>
+
+  <export>
+    <build_type>ament_cmake</build_type>
+  </export>
+</package>
```
Comment (Karsten1987) [inline ros2_control/package.xml]: missing cr

File: .travis.yml [hunk 2]
```diff
@@ -6,7 +6,12 @@ install:
 
 matrix:
   include:
-    - env: JOB_TYPE=dashing
+    # - env: JOB_TYPE=crystal
+    #   script: .ros2ci/travis.bash $JOB_TYPE
+    # Uncomment the following to test against ROS 2 Dashing
+    # - env: JOB_TYPE=dashing
+    #   script: .ros2ci/travis.bash $JOB_TYPE
```
Comment (Karsten1987) [inline .travis.yml]: why removing dashing?

Comment (piraka9011) [reply .travis.yml]: This was initially targeting eloquent, but I think it makes more sense to use nightly. (Unless there's another reason we want to target dashing).

**Pull request merged**